packages/docs/src/hooks/use-showcase.ts

```diff
@@ -51,17 +75,18 @@ export const useShowcaseDemos = () => {
         })
       );
       
-      const demos = resourceRecords
+      const demos: ShowcaseDemo[] = resourceRecords
         .filter((resource): resource is AppResource => !!resource)
         .filter(({ resource }) => SHOWCASE_APPS.includes(resource.id.toString()))
         .map(({ resource, record }) => {
-          return {
-            id: resource.id.toString(),
-            // TODO(wittjosiah): Display name vs. description.
-            title: record.meta.description ?? 'Missing name',
-            description: resource.id.toString() ?? 'Missing description',
+          const id = resource.id.toString();
+          return { // TODO(zarco): Add a `imageHash` property for record inside `data`.
+            id,
+            title: record.data.displayName ?? id,
+            description: record.meta.description ?? '', // use faker sentense
             location: `${BASE_URL}${CID.from(record.data.hash).toString()}`,
-            tags: record.data.keywords ?? []
+            tags: record.data.keywords ?? [],
+            image: record.data.imageDataUri ?? getDefaultImage(record.data.hash)
```

Comment:
Instead of storing the image uri in the record, just assume that an image will be uploaded to ipfs: `${BASE_URL}${cid}/preview.png`.

Reply:
It's a really good approach, we would need to define that in the tutorial so we always get that preview image. If they don't have it we can use one from the gallery.